### PR TITLE
Add TLS protocol

### PR DESCRIPTION
**Pull request by clofresh**
Also, don't log the auth token in the command invocation.

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -73,5 +76,4 @@ if [ -z "$NGROK_PORT" ]; then
 fi
 ARGS="$ARGS `echo $NGROK_PORT | sed 's|^tcp://||'`"
 
-set -x
```
Comment (wernight) [inline entrypoint.sh]: Why remove this? The goal is to show the command being executed.

**Comment by wernight** [comment]
Forgot to ask why `TLS` naming instead of `HTTPS`? Here I think it's actually `HTTPS` you mean (i.e. HTTP-within-SSL/TLS).

**Comment by aaabramov** [comment]
> This avoids logging the token in the console, but it'd still show in a even more likely place: in `ps aux`. That can only be nicely fixed by ngrok supporting passing that token via stdin/file.
> 
> It seems to me that showing the command line helps debug without really reducing the security. I'm fine either way, but consider reverting this change.

Well actually you can store auth token in a file at `~/.ngrok2/ngrok.yml`.

From ngrok docs:

> You must specify your authtoken to ngrok so that your client is tied to this account. ngrok saves your authtoken in ~/.ngrok2/ngrok.yml so that you don't need to repeat this step.

**Comment by aaabramov** [comment]
Created separate issue for that: https://github.com/wernight/docker-ngrok/issues/36

**Comment by labithiotis** [comment]
@wernight will this be merged any time soon?

**Review by wernight — APPROVED** [review]

**Comment by wernight** [comment]
Merged.

**Pull request closed without merging**